src/file/SDL_iostream.c

```diff
@@ -558,6 +559,22 @@ SDL_IOStream *SDL_IOFromFile(const char *file, const char *mode)
             }
             return SDL_IOFromFP(fp, 1);
         }
+    } else if (SDL_strstr(file, "content://") == file) {
```

Comment:
This would be more efficient as SDL_strncmp()

Reply:
Done.